specification/hybridcompute/resource-manager/Microsoft.HybridCompute/stable/2019-12-12/HybridCompute.json

```diff
@@ -0,0 +1,1255 @@
+                    "$ref":"#/definitions/ErrorResponse"
+                 }
+              }
+           },
+           "x-ms-examples":{
+           }
+        },
+        "delete":{
+           "tags":[
+              "machines"
+           ],
+           "operationId":"Machines_Delete",
+           "description":"The operation to remove a hybrid machine identity in Azure.",
+           "parameters":[
+              {
+                 "$ref":"#/parameters/ApiVersionParameter"
+              },
+              {
+                 "$ref":"#/parameters/SubscriptionIdParameter"
+              },
+              {
+                 "name":"resourceGroupName",
+                 "in":"path",
+                 "required":true,
+                 "type":"string",
+                 "description":"The name of the resource group."
+              },
+              {
+                 "name":"name",
+                 "in":"path",
+                 "required":true,
+                 "type":"string",
+                 "description":"The name of the hybrid machine."
+              }
+           ],
+           "responses":{
+              "200":{
+                 "description":"OK"
+              },
+              "204":{
+                 "description":"No Content"
+              },
+              "default":{
+                 "description":"Error response describing why the operation failed.",
+                 "schema":{
+                    "$ref":"#/definitions/ErrorResponse"
+                 }
+              }
+           },
+           "x-ms-examples":{
+           }
+        },
+        "get":{
+           "tags":[
+              "machines"
+           ],
+           "operationId":"Machines_Get",
+           "description":"Retrieves information about the model view or the instance view of a hybrid machine.",
+           "parameters":[
+              {
+                 "$ref":"#/parameters/ApiVersionParameter"
+              },
+              {
+                 "$ref":"#/parameters/SubscriptionIdParameter"
+              },
+              {
+                 "name":"resourceGroupName",
+                 "in":"path",
+                 "required":true,
+                 "type":"string",
+                 "description":"The name of the resource group."
+              },
+              {
+                 "name":"name",
+                 "in":"path",
+                 "required":true,
+                 "type":"string",
+                 "description":"The name of the hybrid machine."
+              },
+              {
+                 "name":"$expand",
+                 "in":"query",
+                 "required":false,
+                 "type":"string",
+                 "description":"The expand expression to apply on the operation.",
+                 "enum":[
+                    "instanceView"
+                 ],
+                 "x-ms-enum":{
+                    "name":"InstanceViewTypes",
+                    "modelAsString":false
```

Comment:
why modelAsString is false? suggest to true to avoid breaking change in SDK side when adding values in future.

Reply:
no idea why this was false. just carried over from the -pr repo. switched.

Reply:
changed 2 other places where this was set to false as well